spec/services/glimr/requests/case_fees_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Glimr::Requests::CaseFees do
+  subject {
+    described_class.new(case_reference: 'TT123', confirmation_code: 1234)
+  }
+
+  describe '#call' do
+    context 'successful' do
+      before do
+        stub_request(:post, %r[/api/tdsapi/requestpayablecasefees$]).
```

Comment:
This fails the tests because `ENV['GLIMR_API_URL']` is not set. We should set the environment variable for the tests, something like `https://www.glimr.com` maybe, and then stub the full URL that comes out of that.